controllers/datafoundationclient.go

```diff
@@ -47,16 +50,29 @@ func dfcSetupWatches(controllerBuilder *builder.Builder) {
 		),
 	)
 	controllerBuilder.
-		Owns(&opv1a1.ClusterServiceVersion{}, csvPredicates).
-		Owns(&ocsclient.StorageClient{}).
-		Owns(&ocsclient.StorageClassClaim{})
+		Owns(&opv1a1.ClusterServiceVersion{}, csvPredicates)
+
+	if utils.MapCRDAvailability(storageClientCRD)[storageClientCRD] {
+		doesCRDExistsAtStart = true
```

Comment:
Is that a global variable? If so it is not differentiated between reconcile calls. We should not do that.